examples/plot_scalable_poly_kernels.py

```diff
@@ -0,0 +1,191 @@
+
+##############################################################################
+# Here we select 5,000 samples for training and 10,000 for testing.
+# To actually reproduce the results in the original Tensor Sketch paper,
+# select 100,000 for training.
+
+X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=5_000,
+                                                    test_size=10_000,
+                                                    random_state=42)
+
+##############################################################################
+# Now scale features to the range [0, 1] to match the format of the dataset in
+# the LIBSVM webpage, and then normalize to unit length as done in the
+# original Tensor Sketch paper [1].
+
+mm = MinMaxScaler().fit(X_train)
+X_train = normalize(mm.transform(X_train))
+X_test = normalize(mm.transform(X_test))
+
+
+##############################################################################
+# As a baseline, train a linear SVM on the original features and print the
+# accuracy. We also measure and store accuracies and training times to
+# plot them latter.
+
+results = {}
+
+lsvm = LinearSVC()
+start = time.time()
+lsvm.fit(X_train, y_train)
+lsvm_time = time.time() - start
+lsvm_score = 100 * lsvm.score(X_test, y_test)
+
+results["LSVM"] = {"time": lsvm_time, "score": lsvm_score}
+print(f"Linear SVM score on raw features: {lsvm_score:.2f}%")
+
+##############################################################################
+# Then we train linear SVMs on the features generated by
+# :class:`PolynomialSampler` with different values for `n_components`, showing
+# that these kernel feature approximations improve the accuracy of linear
+# classification. Note that, since the original samples have 54 features,
+# the explicit feature map of the polynomial kernel of degree four would
+# have approximately 8.5 million features (precisely, 54^4). Thanks to
+# :class:`PolynomialSampler`, we can condense most of the discriminative
+# information of that feature space into a much more compact representation.
+# We repeat the experiment 5 times to compensate for the stochastic nature
+# of :class:`PolynomialSampler`.
+
+n_runs = 3
+for n_components in [250, 500, 1000, 2000]:
+
+    ps_lsvm_time = 0
+    ps_lsvm_score = 0
+    for _ in range(n_runs):
+
+        pipeline = Pipeline(steps=[("kernel_approximator",
+                                    PolynomialSampler(
+                                        n_components=n_components,
+                                        degree=4)),
+                                   ("linear_classifier", LinearSVC())])
+
+        start = time.time()
+        pipeline.fit(X_train, y_train)
+        ps_lsvm_time += time.time() - start
+        ps_lsvm_score += 100 * pipeline.score(X_test, y_test)
+
+    ps_lsvm_time /= n_runs
+    ps_lsvm_score /= n_runs
+
+    results[f"LSVM + PS({n_components})"] = \
+        {"time": ps_lsvm_time, "score": ps_lsvm_score}
```

Comment:
We try to avoid `\` unless there is no choice,
```suggestion
    results[f"LSVM + PS({n_components})"] = {
        "time": ps_lsvm_time, "score": ps_lsvm_score
    }
```